package.json

```diff
@@ -5,8 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "CONTRACT_MAPPING_FILE_PATH=test/erc20/contract_mapping/contract_mapping.json LOG_LEVEL=error CONTRACT_MODE=extract_exact_overwrite mocha --recursive --reporter spec",
-    "lint": "eslint '**/*.js'",
-    "start": "micro"
```

Comment:
Isn't it better to still have the "start" script, just by using ```node index.js```? Or do you want to escape NPM in addition to micro?

Reply:
This wouldn't work, at least without further tweaks, look at how processes look:

``
 PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND                                                                                                                                                                          
     20 root      20   0   11.5g  69764   6404 S   0.3   0.1  28:54.91 node                                                                                                                                                                             
      1 root      20   0    2400     80      0 S   0.0   0.0   0:00.00 sh                                                                                                                                                                               
      8 root      20   0  786312  30792      0 S   0.0   0.0   0:00.41 npm start 
      ```
      
Kubernetes would send the signal to the process with PID 1, this needs to be either `sh` (which would transfer it), or `node`